frontend/src/App.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
-function App() {
+import { Switch, Route } from 'react-router-dom';
+import SignInPage from './components/Pages/SignInPage';
+import GameContextProvider from './context/GameContextProvider';
+import GamePage from './components/Pages/GamePage';
+
+export default function App() {
   return (
-    <div className="App">
-      Schätzmeisterin
-    </div>
+    <GameContextProvider>
+      <Switch>
+        <Route path={'/game/:gameid'} component={GamePage} />
+        <Route path={'/signin/:gameid?'} component={SignInPage} />
```

Comment:
why ?
```suggestion
        <Route path={'/signin/:gameid'} component={SignInPage} />
```

Reply:
This is, so signing in without an id in the path starts a new game. Without the '?' it does not render the page when the path is '.../signin/'. Do you see a better solution for this?

Reply:
ok cool. I did not know that